src/System.Private.CoreLib/src/System/Threading/ManagedThreadId.cs

```diff
@@ -13,109 +13,97 @@
 using System.Runtime;
 using System.Runtime.InteropServices;
 using System.Security;
-
-#pragma warning disable 0420
-
+using System.Collections.Generic;
```

Comment:
Yes this was needed when I used the List and then I forgot to remove it when we changed that